package.json

```diff
@@ -8,7 +8,8 @@
     "react-dev": "webpack -d --watch",
     "build": "webpack -p",
     "start": "node server/index.js",
-    "seed": "mysql -u root -p --host=staybnb.c1bo9omfdnqb.us-east-2.rds.amazonaws.com reviews",
+    "seed": "mysql -u root -p < database/schema.sql",
+    "postseed": "mysql -u root -p < database/data.sql",
```

Comment:
What is the purpose of postseed?

Reply:
For any script name, a script of the same name with `pre` prepended will be run before it, and one with `post` prepended will be run after it. So `postseed` is automatically executed after `seed` when I run `npm run seed` on the command line. [Here's an article about it.](http://www.marcusoft.net/2015/08/pre-and-post-hooks-for-npm-scripting.html)

It is basically the same as if I had written `"mysql -u root -p < database/schema.sql && mysql -u root -p < database/data.sql"`. I've tested that running `npm run seed` does successfully create and populate my database tables.